### PR TITLE
First attempt at publishing 'fakecbed' on conda-forge.

### PR DESCRIPTION
**Pull request by mrfitzpa**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mrfitzpa** [comment]
@conda-forge/help-python, ready for review. Please note that the Windows build did not succeed because `fakecbed` depends on `pytorch`.

**Comment by ocefpaf** [comment]
> @conda-forge/help-python, ready for review. Please note that the Windows build did not succeed because `fakecbed` depends on `pytorch`.

That is fine. Can you please address https://github.com/conda-forge/staged-recipes/pull/28357#issuecomment-2499344515 ?

File: recipes/fakecbed/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 6efb82fc897f1a2095dd28156740b773d87ded90546fba4edd88a2a0994054e9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python 3.8
```
Comment (ocefpaf) [inline recipes/fakecbed/meta.yaml:19]:
```suggestion
   - python {{ python_min }}
```

File: recipes/fakecbed/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - setuptools
+    - wheel
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8
```
Comment (ocefpaf) [inline recipes/fakecbed/meta.yaml:25]:
```suggestion
    - python >={{ python_min }}
```

File: recipes/fakecbed/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - distoptica >=0.1.1
+    - hyperspy
+
+test:
+  imports:
+    - fakecbed
+  commands:
+    - pip check
+  requires:
+    - python 3.8
```
Comment (ocefpaf) [inline recipes/fakecbed/meta.yaml:35]:
```suggestion
    - python {{ python_min }}
```

**Comment by mrfitzpa** [comment]
@ocefpaf Done. Ready to review again. So sorry about the inconvenience. I should have read the linter instructions more carefully.

**Pull request merged**